src/http/validations/device.js

```diff
@@ -13,6 +14,19 @@ const getSingleDevice = (req, res, next) => {
   return next();
 };
 
-export default {
-  getSingleDevice,
+const updateDevice = (req, res, next) => {
+  const { deviceId } = req.params;
+
+  const fieldErrors = new FieldErrors();
+
+  if (!db.validResourceId(deviceId)) fieldErrors.addError('deviceId', 'deviceId should be a ');
```

Comment:
The message here is not complete: `deviceId should be a `???

Reply:
fixed. 
``
deviceId id not valid
```